spec/classes/gmond_spec.rb

```diff
@@ -95,4 +97,19 @@
         with_content(/^\s*ttl\s+=\s+1/)
     end
   end
+
+  context 'with gmond python module' do
```

Comment:
I'd probably call this something like `with gmond_package_name` since that is the parameter being tested.  The rspec coverage in this module is pretty weak...